Gopkg.toml

```diff
@@ -32,7 +32,7 @@ ignored = ["github.com/uber/cadence/.gen"]
 
 [[constraint]]
   name = "github.com/apache/thrift"
-  version = "0.9.3"
+  version = "0.10.0"
```

Comment:
Because client has updated to that version: https://github.com/uber-go/cadence-client/blob/master/Gopkg.toml#L62
dep ensure -update will not work without resolving this conflict

Reply:
Also need to update client because it needs latest client to be able to get children info from describeWF API